ovs/extensions/healthcheck/generic_hc.py

```diff
@@ -86,6 +86,27 @@ def get_log_files_by_path(start_path, recursive=True):
             result_handler.success('All log files are ok!')
 
     @staticmethod
+    @expose_to_cli('ovs','port-ranges-test',HealthCheckCLIRunner.ADDON_TYPE)
```

Comment:
Spaces after each comma
Use PyCharm PEP8 rules for code guidelines

`'ovs'` should be replaced by `MODULE`